ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/rules/NoEmptyFileRuleTest.kt

```diff
@@ -0,0 +1,59 @@
+package com.pinterest.ktlint.ruleset.standard.rules
+
+import com.pinterest.ktlint.ruleset.standard.rules.NoEmptyFileRule.Companion.NO_EMPTY_FILE_PROPERTY
+import com.pinterest.ktlint.test.KtLintAssertThat.Companion.assertThatRule
+import org.junit.jupiter.api.Test
+
+class NoEmptyFileRuleTest {
+    private val noEmptyFileRuleAssertThat = assertThatRule { NoEmptyFileRule() }
+
+    @Test
+    fun `Given not empty kotlin file then ignore the rule for this file`() {
+        val code =
+            """
+            package tmp
+            """.trimIndent()
+        noEmptyFileRuleAssertThat(code)
+            .asFileWithPath("/some/path/Tmp.kt")
+            .withEditorConfigOverride(NO_EMPTY_FILE_PROPERTY to true)
+            .hasNoLintViolations()
+    }
+
+    @Test
+    fun `Given an empty kotlin file then do a return lint error`() {
+        val fileName = "Tmp.kt"
+        val code =
+            """
+
+            """.trimIndent()
```

Comment:
Use a constant `const val EMPTY_FILE = ""`. The newline at lines 26 and 27 will be removed by the `.trimIndent()` anyways so it does not add value to write it like this.